test/irb/test_init.rb

```diff
@@ -163,6 +163,26 @@ def test_use_autocomplete_environment_variable
       IRB.conf[:USE_AUTOCOMPLETE] = orig_use_autocomplete_conf
     end
 
+    def test_copy_command_environment_variable
+      orig_copy_command_env = ENV['IRB_COPY_COMMAND']
+      orig_copy_command_conf = IRB.conf[:COPY_COMMAND]
+
+      ENV['IRB_COPY_COMMAND'] = nil
+      IRB.setup(__FILE__)
+      refute IRB.conf[:COPY_COMMAND]
+
+      ENV['IRB_COPY_COMMAND'] = ''
+      IRB.setup(__FILE__)
+      assert_equal('', IRB.conf[:COPY_COMMAND])
```

Comment:
Unsure what the behavior here should be. Setting this env var overrides the other clipboard programs we use, so right now this would cause `copy` to stop working.

However, we could just as easily interpret `''` to mean "fall back to `pbcopy`".